docs/tutorials/estimator_vqe.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Should we mention that the user need to install qiskit-nature package since it's a separate package. Users followed the quick start guide will not install this package yet: https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-quickstart</p><p> </p><p>In the next cell I have an error of PySCF not installed. So I would suggest mentioning the following as the first step of this tutorial.</p><p> </p><pre class="ql-syntax" spellcheck="false">pip install qiskit-nature
pip install 'qiskit-nature[pyscf]'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Added